library/std/convert.qs

```diff
@@ -264,4 +264,20 @@ namespace Microsoft.Quantum.Convert {
         );
     }
 
+    /// # Summary
+    /// Produces string representation of given double with desired precision (performs rounding).
+    ///
+    /// # Input
+    /// ## input
+    /// Double to be converted.
+    /// ## precision
+    /// Number of decimal digits.
```

Comment:
It's important to clarify that the number is non-negative, since otherwise the user doesn't know that. (We could, for example, replace last digits before the decimal point with 0 for negative precision)
```suggestion
    /// Non-negative number of digits after the decimal point.
```